### DISCUSSION TITLE
Can't grow a geode room without budding amethyst

### DISCUSSION THREAD

**Discussion by coinmansam** [eccentricdevotion/TARDIS#756] [Q&A] [answered]
Like the title says, I can't grow a geode room without budding amethyst, which is unobtainable through survival without the /give command or being able to switch to creative mode. At least that's as far as I know, although if someone knows a way that I can get it besides those, I would love to know.

**Comment by eccentricdevotion** [comment, marked as answer]
[679221e](https://github.com/eccentricdevotion/TARDIS/commit/679221e429d4961f0fee1ddde7285c704e2495c6)
Should be fixed with the commit above.
Grab a new build - https://github.com/eccentricdevotion/TARDIS/releases/tag/1.20.2

**Reply by coinmansam** [reply]
Thank you!
